streams/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/rule/condition/expression/GroovyExpression.java

```diff
@@ -65,6 +65,14 @@ protected String getOperator(Operator operator) {
                     return " >= ";
                 case LESS_THAN_EQUALS_TO:
                     return " <= ";
+                case MULTIPLY:
+                    return "*";
```

Comment:
nit: May be safer to wrap it with space like above operators. Also applies DIVIDE, PLUS, MINUS.

Reply:
nice observation, will do it.